docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[data-views-in-sec]]
+= {Data-sources} in Elastic Security
```

Comment:
I think the content under "The default data view" section would be good to include here it also talks about active data views.

Reply:
For now I tried adding an admonition to the top to direct readers to the info about the default. Let me know if you think we should do more!